src/twisted/conch/test/test_cftp.py

```diff
@@ -1475,36 +1475,25 @@ def _getAttrs(self, s):
         self.patch(FileTransferForTestAvatar, "_getAttrs", _getAttrs)
         self.server.factory.expectedLoseConnection = True
 
-        # PubkeyAcceptedKeyTypes does not exist prior to OpenSSH 7.0 so we
-        # first need to check if we can set it. If we can, -V will just print
-        # the version without doing anything else; if we can't, we will get a
-        # configuration error.
-        d = getProcessValue("ssh", ("-o", "PubkeyAcceptedKeyTypes=ssh-dss", "-V"), env)
-
-        def hasPAKT(status):
-            if status == 0:
-                args = ("-o", "PubkeyAcceptedKeyTypes=ssh-dss")
-            else:
-                args = ()
-            # Pass -F /dev/null to avoid the user's configuration file from
-            # being loaded, as it may contain settings that cause our tests to
-            # fail or hang.
-            args += (
-                "-F",
-                "/dev/null",
-                "-o",
-                "IdentityFile=dsa_test",
-                "-o",
-                "UserKnownHostsFile=kh_test",
-                "-o",
-                "HostKeyAlgorithms=ssh-rsa",
-                "-o",
-                "Port=%i" % (port,),
-                "-b",
-                fn,
-                "testuser@127.0.0.1",
-            )
-            return args
+        # Pass -F /dev/null to avoid the user's configuration file from
+        # being loaded, as it may contain settings that cause our tests to
+        # fail or hang.
+        args = (
+            "-F",
+            "/dev/null",
+            "-o",
+            "IdentityFile=rsa_test",
+            "-o",
+            "UserKnownHostsFile=kh_test",
+            "-o",
+            "HostKeyAlgorithms=ssh-rsa",
+            "-o",
+            "Port=%i" % (port,),
+            "-b",
+            fn,
+            "testuser@127.0.0.1",
+        )
+        return args
```

Comment:
This was a mistake (and causes the slight reduction in coverage), but when I remove this line the test tries to use my normal keys and prompts me for a passphrase, which is clearly wrong.  I'll figure this out soon.